x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LocalLogicalPlanOptimizer.java

```diff
@@ -152,7 +152,7 @@ else if (plan instanceof Project project) {
                         // otherwise point to it
                         else {
                             // since avoids creating field copies
-                            projection = new Alias(f.source(), f.name(), f.qualifier(), nullAlias.toAttribute(), f.id());
+                            projection = new Alias(f.source(), f.name(), null, nullAlias.toAttribute(), f.id());
```

Comment:
Is there no c'tor excluding the qualifier?

Reply:
Added one!